### PR TITLE
xds: add temporary logging to LRS

### PR DESCRIPTION
**Pull request by menghanl**

### CODE REVIEW

File: xds/internal/balancer/lrs/lrs.go [hunk 1]
```diff
@@ -313,9 +313,12 @@ func (ls *lrsStore) ReportTo(ctx context.Context, cc *grpc.ClientConn, clusterNa
 			grpclog.Warningf("lrs: failed to create stream: %v", err)
 			continue
 		}
-		if err := stream.Send(&lrspb.LoadStatsRequest{
+		grpclog.Infof("lrs: created LRS stream")
+		req := &lrspb.LoadStatsRequest{
 			Node: node,
-		}); err != nil {
+		}
```
Comment (easwars) [inline xds/internal/balancer/lrs/lrs.go:319]: Nit: Could this be in a single line.

File: xds/internal/client/client_loadreport.go [hunk 2]
```diff
@@ -44,9 +44,11 @@ func (c *Client) ReportLoad(server string, clusterName string, loadStore lrs.Sto
 		cc      *grpc.ClientConn
 		closeCC bool
 	)
+	c.logger.Infof("Starting load report to server: %s", server)
```
Comment (easwars) [inline xds/internal/client/client_loadreport.go:47]: This doesn't have "lrs" prefix?

File: xds/internal/client/client_loadreport.go [hunk 3]
```diff
@@ -44,9 +44,11 @@ func (c *Client) ReportLoad(server string, clusterName string, loadStore lrs.Sto
...
 	if server == "" || server == c.opts.Config.BalancerName {
 		cc = c.cc
 	} else {
+		c.logger.Infof("LRS server is different from xDS server, starting a new ClientConn")
```
Comment (easwars) [inline xds/internal/client/client_loadreport.go:51]: Same here? Is this expected?

File: xds/internal/balancer/lrs/lrs.go [hunk 4]
```diff
@@ -366,9 +370,11 @@ func (ls *lrsStore) sendLoads(ctx context.Context, stream lrsgrpc.LoadReportingS
 		case <-ctx.Done():
 			return
 		}
-		if err := stream.Send(&lrspb.LoadStatsRequest{
+		req := &lrspb.LoadStatsRequest{
 			ClusterStats: ls.buildStats(clusterName),
-		}); err != nil {
+		}
```
Comment (easwars) [inline xds/internal/balancer/lrs/lrs.go:375]: Same here. Single line.

File: xds/internal/balancer/lrs/lrs.go (hunk 1 above)
Comment (menghanl) [reply xds/internal/balancer/lrs/lrs.go:319]: Done

File: xds/internal/balancer/lrs/lrs.go (hunk 4 above)
Comment (menghanl) [reply xds/internal/balancer/lrs/lrs.go:375]: Done

File: xds/internal/client/client_loadreport.go (hunk 3 above)
Comment (menghanl) [reply xds/internal/client/client_loadreport.go:51]: Same as above.

File: xds/internal/client/client_loadreport.go (hunk 2 above)
Comment (menghanl) [reply xds/internal/client/client_loadreport.go:47]: This is logging with the logger, from the xds_client. It's going to have prefix "xds-client".

This is where things are getting inconsistent. I will need to refactor LRS to support multiple clusters. The logs in this PR are temporary.

**Pull request merged**